### PR TITLE
SE-892: rebalancer

### PR DESCRIPTION
**Pull request by slemasne**

### CODE REVIEW

**Comment by rizwansaeed** [comment]
I think this is better located in https://github.com/finbourne/sample-notebooks/tree/master/examples/use-cases/risk-and-performance

File: examples/use-cases/position-management/Rebalancing with a model portfolio.ipynb [hunk 1]
```diff
@@ -0,0 +1,1790 @@
+{
```
Comment (rizwansaeed) [inline examples/use-cases/position-management/Rebalancing with a model portfolio.ipynb:1]: <p>could you update the index, details here https://github.com/finbourne/sample-notebooks/tree/master/docgen</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (rizwansaeed) [inline examples/use-cases/position-management/Rebalancing with a model portfolio.ipynb:1]: <p>0 index?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (rizwansaeed) [inline examples/use-cases/position-management/Rebalancing with a model portfolio.ipynb:1]: <p>is it easier/better to just delete the portfolio?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

**Comment by slemasne** [comment]
> I think this is better located in https://github.com/finbourne/sample-notebooks/tree/master/examples/use-cases/risk-and-performance

Updated.

File: examples/use-cases/position-management/Rebalancing with a model portfolio.ipynb (hunk 1 above)
Comment (slemasne) [reply examples/use-cases/position-management/Rebalancing with a model portfolio.ipynb]: Updated to delete

Comment (slemasne) [reply examples/use-cases/position-management/Rebalancing with a model portfolio.ipynb]: Updated to correct numbering

Comment (slemasne) [reply examples/use-cases/position-management/Rebalancing with a model portfolio.ipynb]: done

**Review by rizwansaeed — APPROVED** [review]

**Pull request merged**